instrumentation/google.golang.org/grpc/otelgrpc/test/interceptor_test.go

```diff
@@ -754,6 +892,68 @@ func TestStreamServerInterceptor(t *testing.T) {
 	}
 }
 
+func TestStreamServerInterceptorEvents(t *testing.T) {
+	testCases := []struct {
+		Name     string
+		NoEvents bool
+	}{
+		{Name: "With events", NoEvents: false},
+		{Name: "No events", NoEvents: true},
```

Comment:
Same here, we should also test with each event type alone.

Reply:
Updated.